src/components/sidebar/routePathView/routePathListTab/routePathListObject.scss

```diff
@@ -9,11 +9,53 @@
     width: 20px;
 }
 
+.shadow {
+    background-color: #80808024;
+}
+
 .item {
     border-bottom: 1.5px solid $lightGrey;
+    position: relative;
+
+    .headerIcon {
+        width: 30px;
+        position: absolute;
+        height: 30px;
+        left: 5px;
+        top: 50%;
+        margin-top: -15px;
+
+        & > div {
+            margin-left: auto;
+            margin-right: auto;
+            margin-top: 7px;
+        }
+
+        .iconBorder {
+            border-style: solid;
+            border-radius: 30px;
+            display: table;
+            border-width: 3px;
+        }
+
+        .nodeIcon {
+            height: 10px;
+            width: 10px;
+            border-radius: 5px;
+            background-color: white;
+        }
 
-    .shadow {
-        background-color: #80808024;
+        .linkIcon {
+            display: block;
+            height: 90px;
+            background-color: #639fd5;
+            width: 5px;
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            margin-left: -2.5px;
+            margin-top: -45px;
```

Comment:
`transform: translate(x, y)` may be a better way to do this. 
```
top 50%;
left: 50%;
margin: 0;
transform: translate(-50%, -50%)
```
should be in the middle
Now also if the height is ever changed, for example to make the link icon taller, it still stays in the middle.

Reply:
thanks, fixed!